internal/cloudinit/cloudinit.go

```diff
@@ -487,6 +503,10 @@ runcmd:
 - 'wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 ` + pluginUsrBinPath + ` || wget --retry-connrefused --tries=10 --waitretry=10 ` + pluginUsrBinPath + `'
 - 'chmod 777 /usr/bin/plugin'
 {{ end }}
+{{ if .AutoInjectionBinaryURI }}
+- 'wget -nv "{{ .AutoInjectionBinaryURI }}/auto-injection-{{ .Platform.OS }}-{{ .Platform.Arch }} -O /usr/bin/auto-injection'
```

Comment:
No retries similar to other `wget` commands?

Reply:
yeah kept similar to split tests here